sample-easydns.conf

```diff
@@ -0,0 +1,9 @@
+{
+	"domain": "XYZ",
+	"token": "XYZ",
+	"key": "XYZ",
+	"endpoint": "http://sandbox.rest.easydns.net",
+	"format": "json",
+	"delay": 5,
+	"ttl": 600
+}
```

Comment:
```suggestion
}

```

Reply:
Add EOF.

Reply:
I'll check that !

Reply:
Can you be more specific how can I implement the End of File ?

Reply:
It's simple, just add a new line at the end of the file. Github complains that's why.

Reply:
Okay fine , I'll do it

Reply:
Done adding !